### PR TITLE
Feature/donut chart

### PR DESCRIPTION
**Pull request by SVonKarajann**
## Description

Implement a DonutChart component for the DS

PR depends on #160 
[Designs](https://app.abstract.com/projects/27ff87f3-129d-472d-a51e-6d9ce0fa3ef0/branches/3f537df7-b688-4d69-a6fc-6247cabf03ae/commits/latest/files/73f8b89a-e08a-4243-a61d-286325e6f23a/layers/B8C21004-423E-4BCA-949A-84A91D06F46A?collectionId=ed2c27f5-d34d-4f39-b70f-7a59b8d56ac1&collectionLayerId=fc722bfb-61b0-4b94-a3d5-73655ca8ee5a&mode=design&selected=root-F0C1E082-A08A-4C85-9BE7-D9C15ED817EA)

## Screenshot

donut-chart

### CODE REVIEW

**Comment by SVonKarajann** [comment]
reminder

File: src/components/Chart/BarChart/components/CustomLabel/CustomLabel.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { LabelProps } from 'recharts';
+import useTheme from 'hooks/useTheme';
+
+const xValueBase = 16;
+const divisor = 2;
+interface CustomLabelProps extends LabelProps {
```
Comment (Christos-Zacharopoulos) [inline src/components/Chart/BarChart/components/CustomLabel/CustomLabel.tsx:7]: Again, change to type as it's mostly preferred in the project.

**Review by Christos-Zacharopoulos — CHANGES_REQUESTED** [review]

**Comment by Christos-Zacharopoulos** [comment]
I don't see any Tooltip in the designs but are we sure that we don't need one?  Maybe it's something to consider as a change for the future.

**Comment by SVonKarajann** [comment]
> I don't see any Tooltip in the designs but are we sure that we don't need one? Maybe it's something to consider as a change for the future.

indeed, we could add a tooltip in the future if needed

**Review by Christos-Zacharopoulos — APPROVED** [review]

**Pull request merged**